src/components/ui/Modal.js

```diff
@@ -0,0 +1,13 @@
+import '@reach/dialog/styles.css'
+
+import { Dialog } from '@reach/dialog'
+import styled from 'styled-components/macro'
+
+const StyledDialog = styled(Dialog)`
+  border-radius: 23px;
+  width: 75%;
+  height: 70%;
```

Comment:
Is the padding correct on the dialog?

Reply:
I looked on the figma and I couldn't find a padding property so I am not too sure how to handle this

Reply:
Messaged you about this! But will include here as well for other ppl:

- A lot of layout CSS generated by Figma isn't too reliable, except for font sizes, shadows, colors, etc. Figma tends to generate absolute top, left, bottom, right measurements, which doesn't reflect that you should use padding
- Your goal is always to try to make your component **visually** similar to the design. Figma's CSS is just a tool to help you achieve that. Right now, it looks like there's too much space between the border and the text. How can you fix that?